src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -396,7 +396,7 @@ End Class
             End Using
         End Function
 
-        <WpfFact>
+        <WpfFact(Skip:="https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1906990")>
```

Comment:
This failed after updating the VS editor package to 17.9. (Only fails in our CI, Spanish VM)
https://github.com/dotnet/roslyn/pull/70471

Test is testing the behavior for both our code & editor side impl of `IAsyncCompletionBroker`.
So, if this is regressed, it's already in VS, since editor team owns the insertion of the dll.
I have talked to Gen and we can temp skip this test and let editor folks to investigate this